website/src/pages/docs/server-side-rendering.mdx

```diff
@@ -66,6 +66,10 @@ const linkTags = extractor.getLinkTags() // or extractor.getLinkElements();
 const styleTags = extractor.getStyleTags() // or extractor.getStyleElements();
 ```
 
+> Note that you must call `renderToString` before collecting the script tags. Otherwise loadable
```

Comment:
in the given example `collectChunks` is used before `renderToString`. 
Sorry, but I am not sure that this note would be helpful and not cause more confusion.

Reply:
Of course, I'm referring to `getScriptTags`, `getLinkTags` and `getStyleTags`. I could maybe be even more explicit so there is no room for confusipn?

I personally spent over a day hitting my head with this issue, and there are issues that report the same confusion (#853 is one). I think a note like this will help a lot of people avoid this issue and better understand how to use loadable.

Reply:
Can a note right after `renderToString`, ie the one "in the example" do a better job?

Reply:
Sure. I added a note in the code listing now, that hopefully is clearer and less obtrusive.